src/common.h

```diff
@@ -63,10 +63,12 @@ const std::vector<std::string> TRITON_RESERVED_REQUEST_PARAMS{
   do {                                                                 \
     TRITONSERVER_Error* err__ = (X);                                   \
     if (err__ != nullptr) {                                            \
-      return TRITONSERVER_ErrorNew(                                    \
+      auto new_err = TRITONSERVER_ErrorNew(                            \
```

Comment:
Future scope / question:

Can we reset the message of the the error instead of creating a new one and deleting the old one?  Maybe requires a new / unwanted api TRITONSERVER_ErrorSetMessage() ....

Reply:
Right, that needs new API.. You can create ticket for it if you want, but I don't see how useful it can be.